core/src/main/resources/error/error-classes.json

```diff
@@ -2082,6 +2117,11 @@
     ],
     "sqlState" : "42704"
   },
+  "UNRESOLVABLE_TABLE_VALUED_FUNCTION" : {
+    "message" : [
+      "could not resolve `<name>` to a table-valued function. Please make sure that `<name>` is defined as a table-valued function and that all required parameters are provided correctly. If `<name>` is not defined, please create the table-valued function before using it. For more information about defining table-valued functions, please refer to the Apache Spark documentation."
```

Comment:
I think the caller side shall pass quoted identifiers, seems we don't need backticks in the error message.